### ISSUE TITLE
Request for clarification of the usage notes on vkCmdCopyBufferToImage

### ISSUE THREAD

**Issue by haasn** [KhronosGroup/Vulkan-Docs#676]
The `Valid Usage` section of `vkCmdCopyBufferToImage` states the following:

>  The buffer region specified by each element of pRegions must be a region that is contained within srcBuffer

However, I'm having trouble answering this question about a corner case: Suppose I have a 64x64 2D image, and I present an upload operation that looks like this:

```
(VkBufferImageCopy) {
    .imageOffset = {0, 0, 0},
    .imageExtent = {2, 2, 1},
    .bufferRowLength = 64,
    ...
}
```

In other words, I'm uploading a 2x2 subregion, with a horizontal stride of 64. What is the minimum buffer size (in texels) for this to be a valid operation? Is it 128 (imageExtent.height * bufferRowLength) since I'm transferring two rows of a 2D image, or is it 66 (2 pixels + 62 pixels of padding + 2 more pixels) since that's the minimum contiguous region that contains all of the pixels to be transferred and no superfluous padding as a result of the stride?

In image processing it's often typical for the padding at the end of the last row of the included, that is, the total size is typically calculated as `height * stride`, rather than `(height - 1) * stride + width`. But it would seem to me that, strictly speaking, this extra padding isn't necessary. Is this correct?

**Comment by krOoze** [comment]
The spec works under the assumption that "region" is the copied part only (and the rest is left untouched).
So I would assume the later.

**Comment by oddhack** [comment]
This should be fixed in the 1.1.74 spec update.
